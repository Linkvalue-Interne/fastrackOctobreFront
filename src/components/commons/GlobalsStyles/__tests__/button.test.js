import React from 'react';
import renderer from 'react-test-renderer';
import serializer from 'jest-emotion';
import Button from '../button';

expect.addSnapshotSerializer(serializer);

describe('button', () => {
  it('should renders with correct styles', () => {
    const mock = {
      colors: {
        primary: '#111',
        white: '#fff',
      },
    };
    const tree = renderer.create(<Button theme={mock} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
