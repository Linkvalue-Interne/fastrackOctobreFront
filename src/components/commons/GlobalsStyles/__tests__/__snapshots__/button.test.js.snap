// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button should renders with correct styles 1`] = `
.emotion-0 {
  background: #111;
  border-radius: 90px;
  padding: .5rem 1rem;
  margin: 0.5rem;
  color: #fff;
  -webkit-transition: -webkit-transform .03s cubic-bezier(0.47,0,0.745,0.715);
  -webkit-transition: transform .03s cubic-bezier(0.47,0,0.745,0.715);
  transition: transform .03s cubic-bezier(0.47,0,0.745,0.715);
  border: none;
}

.emotion-0:hover {
  -webkit-transform: scale(1.05);
  -ms-transform: scale(1.05);
  transform: scale(1.05);
}

<button
  className="emotion-0"
/>
`;
